cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -726,7 +758,29 @@ public KafkaAssemblyOperator(Vertx vertx, boolean isOpenShift,
         }
 
         Future<ReconciliationState> kafkaAncillaryCm() {
```

Comment:
This method has essentially the same code as the `zookeeperAncillaryCm()` method: You should try to factor out a common method.